cg/services/analysis_starter/configurator/file_creators/nextflow/params_file/utils.py

```diff
@@ -0,0 +1,32 @@
+import copy
+import re
+
+
+def replace_values_in_params_file(workflow_parameters: dict) -> dict:
```

Comment:
I copied these functions from the analysisAPI, they should be refatored in the near future but right now they are not used